Options/WeakAurasOptions.lua

```diff
@@ -7375,15 +7428,6 @@ function WeakAuras.CreateFrame()
   codereviewCancel:SetWidth(100);
   codereviewCancel:SetText(L["Okay"]);
 
-  function codereview.Close(self)
```

Comment:
For some reason there were two codereview.Close() functions. There should be one left just a few lines later.